packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -101,11 +101,14 @@ API endpoints that take username and password credentials, including the [Authen
   **Endpoints**
   * `/api/v1/authn/factors`
   * `/api/v1/users/${userId}/factors`
+  * `/user/profile/mobile_phone`
```

Comment:
This is not a public endpoint for customers, even though we rate limit it, but it is triggered through V1 UI

Reply:
I removed this endpoint.